sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/FaceSessionClient.java

```diff
@@ -1312,58 +757,40 @@ public BinaryData getSessionImage(String sessionImageId) {
-     * more details.
-     *
-     * @param parameters Liveness with verify session parameter.
-     * @param verifyImage Verify image to be submitted during the session creation.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
-     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
-     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
-     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return response of liveness session with verify creation with verify image provided.
-     */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public CreateLivenessWithVerifySessionResult
-        createLivenessWithVerifySession(CreateLivenessWithVerifySessionContent parameters, BinaryData verifyImage) {
-        if (verifyImage == null) {
-            return createLivenessWithVerifySession(parameters);
-        }
-        VerifyImageFileDetails verifyImageFileDetails = new VerifyImageFileDetails(verifyImage);
-        CreateLivenessWithVerifySessionMultipartContent realParameters
-            = new CreateLivenessWithVerifySessionMultipartContent(parameters, verifyImageFileDetails);
-        return this.createLivenessWithVerifySessionWithVerifyImage(realParameters);
+        return createLivenessWithVerifySessionWithResponse(new MultipartFormDataHelper(requestOptions)
+            .serializeTextField("livenessOperationMode", Objects.toString(body.getLivenessOperationMode()))
+            .serializeTextField("deviceCorrelationIdSetInClient",
+                Objects.toString(body.isDeviceCorrelationIdSetInClient()))
+            .serializeTextField("enableSessionImage", Objects.toString(body.isEnableSessionImage()))
+            .serializeTextField("livenessModelVersion", Objects.toString(body.getLivenessModelVersion()))
+            .serializeTextField("returnVerifyImageHash", Objects.toString(body.isReturnVerifyImageHash()))
+            .serializeTextField("verifyConfidenceThreshold", Objects.toString(body.getVerifyConfidenceThreshold()))
+            .serializeFileField("verifyImage", body.getVerifyImage().getContent(),
+                body.getVerifyImage().getContentType(), body.getVerifyImage().getFilename())
+            .serializeTextField("deviceCorrelationId", body.getDeviceCorrelationId())
+            .serializeTextField("authTokenTimeToLiveInSeconds",
+                Objects.toString(body.getAuthTokenTimeToLiveInSeconds()))
+            .end()
+            .getRequestBody(), requestOptions).getValue().toObject(LivenessWithVerifySession.class);
     }
```

Comment:
This method is removed by me. It seems no longer needed, as the `CreateLivenessWithVerifySessionContent` parameter already have `verifyImage` within it.